generate_fixtures.py

```diff
@@ -228,16 +235,23 @@ def __init__(self):
         level_1_delegation = self.repository.targets._delegated_roles.get('unclaimed')
 
         # Delegate from level_1_delegation to level_1 target-signing key
-        (public_level_2_error_key, private_level_2_error_key) = self.write_and_import_keypair(
+        (public_key, private_key) = self.write_and_import_keypair(
```

Comment:
Why did these variables get renamed?

Reply:
I wasn't sure there was point in having separate keys for every role since what are testing if the resources throw a not found exception

Reply:
Reverted this an create a new key for the new delegation.